lib/widgets/app.dart

```diff
@@ -21,7 +22,12 @@ class ZulipApp extends StatelessWidget {
       // Note that specifiying only "Noto Color Emoji" in the fallback list,
       // Flutter tries to use it to draw even the non emoji characters
       // which leads to broken text rendering.
-      fontFamilyFallback: const <String>['sans-serif', 'Noto Color Emoji'],
+      fontFamilyFallback: [
+        // …since apparently iOS doesn't support 'sans-serif', use this instead:
+        //   https://github.com/flutter/flutter/issues/63507#issuecomment-1698504425
+        if (Platform.isIOS) '.AppleSystemUIFont' else 'sans-serif',
```

Comment:
This should be the Flutter "target platform", I think, rather than the actual current platform. Cf f669f16f820350aa78f24e403a8f782dad5bbfda.